pages/home.jsx

```diff
@@ -1,19 +1,157 @@
+var moment = require('moment');
 
 var HeroUnit = require('../components/hero-unit.jsx');
 var Blockquote = require('../components/blockquote.jsx');
 var Illustration = require('../components/illustration.jsx');
+var IconLinks = require('../components/icon-links.jsx');
+var IconLink = require('../components/icon-link.jsx');
+
+var config = require('../lib/config');
 
 var CaseStudies = React.createClass({
   render: function() {
     return (
-      <div className="case-studies">
-        <Blockquote className="primary-quote" author="Maurya C. New York, United States"
-            imgSrc="/img/pages/home/maurya-nyc.png" imgSrc2x="/img/pages/home/maurya-nyc@2x.png" imgAlt="Maurya NYC Quote">
-          <p>Web literacy is about more than coding - it's about how you can be a better web citizen.</p>
-        </Blockquote>
+      <div className="row">
+        <div className="col-sm-offset-2 col-sm-8 col-md-offset-2 col-md-8 col-lg-offset-2 col-lg-8">
+          <div>
+            <Blockquote author="Maurya C. New York, United States"
+                imgSrc="/img/pages/home/maurya-nyc.png" imgSrc2x="/img/pages/home/maurya-nyc@2x.png">
+              <p>"Web literacy is about more than coding - it's about how you can be a better web citizen."</p>
+            </Blockquote>
+          </div>
+        </div>
+      </div>
+    );
+  }
+});
+
+var FeaturedPost = React.createClass({
+  propTypes: {
+    data: React.PropTypes.object.isRequired
+  },
+  render: function() {
+    var parsedDate = moment(this.props.data.publishedDate);
+    return(
+      <div className="featured-post">
+        <div className="entry-posted-container">
+          <p className="entry-posted">
+            <time className="published" title={this.props.data.publishedDate} dateTime={this.props.data.publishedDate} >
+              <span className="posted-month">{parsedDate.format("MMM")}</span>
+              <span className="posted-date">{parsedDate.format("DD")}</span>
+              <span className="posted-year">{parsedDate.format("YYYY")}</span>
+            </time>
+          </p>
+        </div>
+        <div className="entry-header-container">
+          <h3 className="entry-title"><a href={this.props.data.link}>{this.props.data.title}</a></h3>
+          <cite className="author">{this.props.data.author}</cite>
+        </div>
+        <p className="excerpt">
+          {this.props.data.contentSnippet}
+        </p>
+        <a className="more" href={this.props.data.link}>Continue reading</a>
+      </div>
+    );
+  },
+});
+
+var LatestPosts = React.createClass({
+  propTypes: {
+    data: React.PropTypes.array.isRequired
+  },
+  render: function() {
+    return (
+      <ul className="recent-posts">
+        {
+          this.props.data.map(function(post, i) {
+            return (
+              <li key={i}>
+                <a className="post-title">{post.title}</a>
+                <time className="published" title={post.publishedDate} dateTime={post.publishedDate}>
```

Comment:
same reason as mentioned in https://github.com/mozilla/teach.webmaker.org/pull/943/files#r31094020